### ISSUE TITLE
Changes in Design and Alignment of Blog Page is needed

### ISSUE THREAD

**Issue by TeekshaHarish** [akshitagupta15june/PetMe#398]
After interacting with the website I have found that the Blog page needs some major changes in designs and alignment.
Firstly, the Read button in many cards in Blogs page comes just after the description of the blog instead of coming under it which needs to be fixed. 

Second ,when we click the READ button, the new page which opens having the detailed blog is misaligned which doesn't look good and needs to be improved. I can work on these issues.

Please assign me this issue under GSSOC'2023

**Comment by akshitagupta15june** [comment]
ok assigning you

**Comment by Nayanika-12** [comment]
Are you working on this @TeekshaHarish ?

**Comment by TeekshaHarish** [comment]
Yes i am @Nayanika-12 ,i have created a PR for this issue and am waiting for its review
